whois-api/src/main/java/net/ripe/db/whois/api/rdap/WhoisRdapService.java

```diff
@@ -209,6 +213,33 @@ protected Response lookupObject(final HttpServletRequest request, final Set<Obje
         return getResponse(request, result);
     }
 
+    protected Response lookupForOrganisation(final HttpServletRequest request, final Set<ObjectType> objectTypes,
+                                             final String key) {
+        Query autnumInetnumForOrganisationQuery = Query.parse(
```

Comment:
Move this variable down past the `organisationResult` check as it's not needed until afterwards